.github/workflows/test_and_deploy.yml

```
@@ -69,7 +69,7 @@ jobs:
 
       - name: Run cellfinder tests
         run: |
-          python -m pytest tests/tests/test_integration/test_detection.py
+          python -m pytest -v
```

Would also recommend
```suggestion
          python -m pytest --color=yes -v
```